apps/docs/src/pages/tabs.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Tabs
+description: Sample Tabs documentation
+layout: ../layouts/MainLayout.astro
+---
+
+import { Tab } from "@selleo/core/src/Tabs";
+
+import Preview from "../components/Preview.astro";
+import Description from "../components/Description.astro";
+
+<Description>
+  Below you can find a tabs component styled with tailwindcss.
+</Description>
+
+<Preview component={Tab} name="Default tab" active={false} />
```

Comment:
default tab in dark mode doesn't have a bg on figma
![image](https://user-images.githubusercontent.com/32620457/221527346-27063422-2bf6-4eb7-b00a-7c9292ee2f5b.png)

Reply:
Default tab shouldn't have bg on both modes